### ISSUE TITLE
ENUM values are always lowercased regardless of actual case

### ISSUE THREAD

**Issue by abmyii** [dolthub/go-mysql-server#1161]
https://github.com/dolthub/go-mysql-server/blob/82a31eff75d627085656462eb549ef45c559563b/sql/settype.go#L102

I'm not sure what the reasoning is behind this, bu it is causing issues with the SQLAlchemy ORM which sees the ENUM values as lowercase, causing it to raise errors like:
```python
'UNSPECIFIED' is not among the defined enum values. Enum name: None. Possible values: inpatient, outpatient, both, unspecified
```

Schema (https://www.dolthub.com/repositories/dolthub/hospital-price-transparency-v3):
```sql
create table prices (
    [...]
    inpatient_outpatient enum('INPATIENT', 'OUTPATIENT', 'BOTH', 'UNSPECIFIED') default 'UNSPECIFIED',
    [...]
);
```

ENUM name is also None which should also be fixed.

**Comment by fulghum** [comment]
Hey @abmyii, thanks for reporting this. We've had a couple customer requests for this now, so I'll see if we can knock this one out in the next week or two as we wrap up some other in-flight issues. 

We've got an existing issue open for this, so I'm going to resolve this one in favor of: https://github.com/dolthub/dolt/issues/3991

Good point about enum name being dropped, too. Thanks for reporting that! I'll mention that in the other issue to make sure we close that gap, too.

**Comment by fulghum** [comment]
Hey @abmyii, just closing the loop on this one. We got the fix for this issue released in [Dolt 0.40.26](https://github.com/dolthub/dolt/releases/tag/v0.40.26).

Thanks for reporting this one!

**Comment by abmyii** [comment]
> Hey @abmyii, just closing the loop on this one. We got the fix for this issue released in [Dolt 0.40.26](https://github.com/dolthub/dolt/releases/tag/v0.40.26).

Great, thank you for the speedy fix!
